std_msgs/Header header 
float32 percentage
float32[] side_lengths
float32[] position
bool is_full